x-pack/platform/plugins/private/monitoring_collection/server/plugin.ts

```diff
@@ -14,12 +14,12 @@ import {
   ServiceStatus,
 } from '@kbn/core/server';
 import { MakeSchemaFrom } from '@kbn/usage-collection-plugin/server';
-import { metrics } from '@opentelemetry/api-metrics';
+import { api, metrics, resources } from '@elastic/opentelemetry-node/sdk';
 import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-grpc';
-import { MeterProvider, PeriodicExportingMetricReader } from '@opentelemetry/sdk-metrics-base';
-import { resourceFromAttributes } from '@opentelemetry/resources';
-import { diag, DiagLogger, DiagLogLevel } from '@opentelemetry/api';
-import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
+import { ATTR_SERVICE_NAME, ATTR_SERVICE_VERSION } from '@opentelemetry/semantic-conventions';
+// import { ATTR_SERVICE_INSTANCE_ID } from '@opentelemetry/semantic-conventions/incubating';
```

Comment:
iirc, the OTel docs actually mention you should copy them over (while they are in development) and not import directly